data/tabular/blood_brain_barrier_martins_et_al/transform.py

```diff
@@ -0,0 +1,235 @@
+Assistant: Ok, this {SMILES__description} is {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}: {SMILES#}""",  # noqa: E501
+            """User: I want to create a molecule {SMILES__description}.
+Assistant: This sounds very exciting! Should it be a special molecule?
+User: Yes, the molecule should {penetrate_BBB#not &NULL}be {penetrate_BBB__names__adjective}.
+Assistant: Got it, this {SMILES__description} is {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}: {SMILES#}""",  # noqa: E501
+            # Benchmarking text templates
+            "Is the {SMILES__description} {SMILES#} {penetrate_BBB__names__adjective}:<EOI> {penetrate_BBB#yes&no}",  # noqa: E501 for the benchmarking setup <EOI> separates input and output
+            # todo: check if we go for multiple choice only and remove the benchmarking template above and below
+            """Task: Please classify a molecule based on the description.
+Description: A molecule that is {penetrate_BBB__names__adjective}.
+Molecule {SMILES__description}: {SMILES#}
+Constraint: Even if you are uncertain, you must pick either "True" or "False" without using any other words.
+Result:<EOI> {penetrate_BBB#False&True}""",
+            """Task: Please answer the multiple choice question.
+Question: Is the molecule with the {SMILES__description} of {SMILES#} {penetrate_BBB__names__adjective}?
+Constraint: Even if you are uncertain, you must pick either {%multiple_choice_enum%2%aA1} without using any other words.
+Options:
+{penetrate_BBB%}
+Answer: {%multiple_choice_result}""",
+            """Task: Please answer the multiple choice question.
+Question: Which molecules are {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}?
+Constraint: You must select none, one or more options from {%multiple_choice_enum%2-5%aA1} without using any other words.
+Options:
+{SMILES%penetrate_BBB%}
+Answer: {%multiple_choice_result}""",  # noqa: E501
+            """Task: Please answer the multiple choice question.
+Question: Is the molecule with the {SMILES__description} of {SMILES#} {penetrate_BBB__names__adjective}?
+Constraint: Even if you are uncertain, you must pick either {%multiple_choice_enum%2%aA1} without using any other words.
+Options:
+{penetrate_BBB%}
+Answer:<EOI> {%multiple_choice_result}""",
+            """Task: Please answer the multiple choice question.
+Question: Which molecules are {penetrate_BBB#not &NULL}{penetrate_BBB__names__adjective}?
+Constraint: You must select none, one or more options from {%multiple_choice_enum%2-5%aA1} without using any other words.
+Options:
+{SMILES%penetrate_BBB%}
+Answer:<EOI> {%multiple_choice_result}""",  # noqa: E501
+        ],
+    }
+
+    def str_presenter(dumper, data):
+        """configures yaml for dumping multiline strings
+        Ref: https://stackoverflow.com/questions/8640959/how-can-i-control-what-scalar-form-pyyaml-uses-for-my-data
```

Comment:
shall we import this from `utils`?

Reply:
I'm pretty sure we have that in a few tabular data setup. I'll check that.

Reply:
Do you know a good setup to handle that?
I didn't find any nice solution except adding the additional import path or moving it to the chemnlp library.